src/test/ui/issues/issue-3044.stderr

```diff
@@ -8,6 +8,12 @@ LL | |     });
    |
    = note: expected closure `[closure@$DIR/issue-3044.rs:3:28: 4:6]`
             found unit type `()`
+help: consider returning local binding `x`
+   |
+LL ~     needlesArr.iter().fold(|x, y| {
+LL +         x
+LL +     
```

Comment:
It seems like we might have one too many newlines 🤔

Reply:
hmmm.. maybe. let me bump this to MachineApplicable and see what it really renders like.